test/known_issues/test-readline-interface-tty-readstream.js

```diff
@@ -0,0 +1,73 @@
+/**
```

Comment:
Can you follow the conventions of the other files in this directory (first line is strict mode, second line is a `Refs:`).